tests/python_tests/test_whisper_pipeline.py

```diff
@@ -114,30 +109,48 @@ def read_whisper_model(params, **tokenizer_kwargs):
     )
 
 
-def compare_genai_and_opt_pipelines(opt_pipe, genai_pipe, dataset_id):
-    ds = datasets.load_dataset(dataset_id, "clean", split="validation")
-    opt_infer_time = 0
-    genai_infer_time = 0
+@dataclass
+class GenerationConfig:
```

Comment:
why WhisperGenerationConfig is not enough here? IMO, it's better to rely on official GenAI API